# Publish all joint states -----------------------------------
    joint_state_controller:
      type: joint_state_controller/JointStateController
      publish_rate: 50

    # Position Controllers ---------------------------------------
    base_waist_joint_position_controller:
      type: effort_controllers/JointPositionController
      joint:  base_waist_joint
      pid: {p: 60.0, i: 10.0, d: 10.0}
    waist_body_joint_position_controller:
      type: effort_controllers/JointPositionController
      joint: waist_body_joint
      pid: {p: 60, i: 15.0, d: 1.0}
    body_head_joint_position_controller:
      type: effort_controllers/JointPositionController
      joint: body_head_joint
      pid: {p: 10, i: 1, d: 0.0}

    joint_state_controller:
      type: joint_state_controller/JointStateController
      publish_rate: 50

    left_wheel_joint_velocity_controller:
      type: velocity_controllers/JointVelocityController
      joint: wheel_left_joint  
      pid: {p: 00.1, i: 0.00001, d: 0.0001}

    right_wheel_joint_velocity_controller:
      type: velocity_controllers/JointVelocityController
      joint: wheel_right_joint 
      pid: {p: 100.0, i: 0.1, d: 10.0}
